app-misc/ola/ola-0.10.7.ebuild

```diff
@@ -0,0 +1,100 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{6,7} )
+
+inherit autotools python-single-r1
+
+if [[ "${PV}" == "9999" ]]; then
+	inherit git-r3
+
+	EGIT_REPO_URI="https://github.com/OpenLightingProject/${PN}"
+fi
+
+DESCRIPTION="Open Lighting Architecture"
+HOMEPAGE="https://www.openlighting.org/"
+
+if [[ "${PV}" != "9999" ]]; then
+	SRC_URI="https://github.com/OpenLightingProject/${PN}/releases/download/${PV}/${P}.tar.gz"
+else
+	SRC_URI=""
+fi
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+IUSE="examples ftdi httpd osc python test usb"
+
+RESTRICT="!test? ( test )"
+
+# Since media-libs/liblo is not KEYWORDed for arm, we force-disable it
+REQUIRED_USE="
+	arm? ( !osc )
+	arm64? ( !osc )
```

Comment:
This is not the correct way. You need to utilize use.masks in profile/arch/

Reply:
Would it be easier to get the package accepted with this optional feature removed? I assume that stuff in `profile/` should not be edited by proxy maintainers?